strings.asm

```diff
@@ -8,21 +8,23 @@
 
 ; ## GENERAL STRINGS
 
-; All general strings must be zero-terminated, names start with "s_",
+; the base36 alphabet for printing values in current base
+
+alpha36:  .text "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+; All general strings are bit-7 terminated (.shift), names start with "s_",
```

Comment:
I dropped the zero terminators to save some bytes and used .shift which flags bit 7 of the last character.   Needed minor change to print_common